packages/esm-appointments-app/src/appointments/common-components/appointments-actions.component.tsx

```diff
@@ -22,26 +22,25 @@ interface AppointmentsActionsProps {
 const AppointmentsActions: React.FC<AppointmentsActionsProps> = ({ appointment }) => {
   const { t } = useTranslation();
   const { checkInButton, checkOutButton } = useConfig<ConfigObject>();
-  const { visits, mutateVisit } = useTodaysVisits();
+  const { visits, mutateVisit } = useTodaysVisits(); // TODO doesn't work if visit didn't start today?  what about inpatient?
   const patientUuid = appointment.patient.uuid;
   const visitDate = dayjs(appointment.startDateTime);
   const hasActiveVisitToday = visits?.some((visit) => visit?.patient?.uuid === patientUuid && visit?.startDatetime);
```

Comment:
We will switch the logic around the "Check In" button as part of O3-3147, leaving this as-is for now.